core/trino-parser/src/main/java/io/trino/sql/tree/QualifiedName.java

```diff
@@ -63,7 +62,7 @@ private QualifiedName(List<Identifier> originalParts)
         this.parts = originalParts.stream()
                 .map(QualifiedName::mapIdentifier)
                 .collect(toImmutableList());
-        this.name = Joiner.on(".").join(parts);
+        this.name = String.join(".", parts);
```

Comment:
Is this just a cleanup? Or part of `Speed up QualifiedName constructor`?

Reply:
The java version of this is indeed faster, but I don't think faster enough to make a difference. So this is more of a cosmetic change